index.js

```diff
@@ -7,18 +7,17 @@ const {
   owner,
   repo,
   githubApiToken,
-  assignments,
-  issuesToPullFrom,
+  numIssuesPerPerson,
+  numIssuesToPickFrom,
   assignees,
   additionalQueryParams,
   labelsToAdd,
+  message,
   dryRun=true // Safer to force you to turn it on
 } = require('./config.json');
 
-// TODO: work out how to make the message generic while allowing for some
-//       templating to occur.
-const assignmentMessage = (ass) =>
-`@${ass} please close or schedule before the end of this sprint. See [triaging old issues](https://github.com/medic/medic-docs/blob/master/md/dev/workflow.md#triaging-old-issues). `;
+const assignmentMessage = (message, assignee) => eval('`'+ message +'`');
```

Comment:
Just to be clear, `eval` is allowing you to do whatever you want in JS on that system, as that user. You could argue _for now_ it doesn't matter because this is something run locally with your own config. However, if you ever wanted to move this into something that ran on a server and was talked to via slack, or whatever, it is untenably insecure.

With that in mind, you may want to consider re-writing this to be much simpler templating, e.g:
```
// exampleMessage = "Hi @@ please fill this out"
message.replace(new RegExp(/@@/, 'g'), `@${assignee}`)
```